File: jsfring-webservice/src/main/java/fr/pinguet62/jsfring/webserivce/dto/RightDto.java
package fr.pinguet62.jsfring.webserivce.dto;

import fr.pinguet62.jsfring.model.sql.Right;

/** @see Right */
public final class RightDto {

    private String code;

    private String title;

    public String getCode() {
        return code;
    }

    public String getTitle() {
        return title;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public void setTitle(String title) {
        this.title = title;
    }

}

File: jsfring-webservice/src/main/kotlin/fr/pinguet62/jsfring/webservice/dto/RightDto.kt
package fr.pinguet62.jsfring.webservice.dto

import fr.pinguet62.jsfring.model.sql.Right

/**
 * @see Right
 */
data class RightDto(
        var code: String?,
        var title: String?
) {
    constructor() : this(null, null)
}
